js/activate-form.js

```diff
@@ -0,0 +1,28 @@
+const adForm = document.querySelector('.ad-form');
+const adFormFieldsets = adForm.querySelectorAll('fieldset');
+const mapForms = document.querySelector('.map__filters');
+const mapFilters = mapForms.querySelectorAll('fieldset', 'select');
+
+const formDisable = () => {
+  adForm.classList.add('ad-form--disabled');
+  adFormFieldsets.forEach((fieldset) => {
+    fieldset.setAttribute('disabled', 'true');
+  });
+  mapForms.classList.add('ad-form--disabled');
+  mapFilters.forEach((form) => {
+    form.setAttribute('disabled', 'true');
```

Comment:
Здесь лучше к свойству обращаться для единообразия
```suggestion
    formElement.disabled = true;
```